res/css/photon/message-bar.css

```diff
@@ -71,6 +90,7 @@
 .photon-message-bar-warning .photon-message-bar-action-button:hover,
 .photon-message-bar-warning .photon-message-bar-close-button:hover:active {
   background-color: var(--yellow-70);
+  color: #fff;
```

Comment:
I found that in the spec, the text color should be white because the background is darker.